sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/CheckstyleRegressionTest.java

```diff
@@ -36,7 +36,10 @@
 
     /** List of checks to suppress if we dynamically add it to the configuration. */
     private static final List<String> ADD_CHECK_SUPPRESSIONS = Arrays
-            .asList("ReturnCountExtendedCheck");
+            .asList(
+                    "EnumTrailingCommaCheck",
```

Comment:
please comment above "// conflicting with other Checks"

Reply:
@romani What was the other check in conflict here? We should probably specifically list it.

Reply:
Done: 038ff42971b6e8c9ebefb28e13c9a5680ef66091

Added the comment directly to the line of `EnumTrailingCommaCheck` and included the checks in conflict as already outlined in [this comment on checkstyle issue 6054](https://github.com/checkstyle/checkstyle/pull/6054#issuecomment-409405662).

Reply:
Can we mark this part of the conversation as resolved?